.travis.yml

```diff
@@ -21,6 +21,9 @@ matrix:
     - php: 7.1
       env:
         - dependencies=lowest
+   - php: 7.2
```

Comment:
Took me a while to spot but this line is missing a space for the `-`, hence the config parse error

Reply:
Yesterday was semicolons, now spaces 😂
Sorry about that!

Reply:
No problem :+1: